types/prosemirror-commands/index.d.ts

```diff
@@ -11,6 +11,7 @@ import { EditorView } from 'prosemirror-view';
 
 export function deleteSelection(state: EditorState, dispatch?: (tr: Transaction) => void): boolean;
 export function joinBackward(state: EditorState, dispatch?: (tr: Transaction) => void, view?: EditorView): boolean;
+export function selectNodeBackward(state: EditorState, dispatch?: (tr: Transaction) => void, view?: EditorView): boolean;
```

Comment:
You should bump the version at the top of the file to whatever major.minor version of ProseMirror added this.

Reply:
Updated!